notebooks/getting_started/demo_bq_dataframes.ipynb

```diff
@@ -0,0 +1,2087 @@
+{
```

Comment:
<p>Sandeep: Can be commented out, just one read_gbq use case is fine </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/1200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Done.